src/public/style/mobile.css

```diff
@@ -0,0 +1,450 @@
+@media (max-width: 600px) {
+section {
+ position: fixed;
+ top: 0;
+ left: 0;
+ width: 100%;
+}
+#visualization{
+height: 300px;
+background: #333 url('../images/hand.png') no-repeat 39px 78px;
```

Comment:
Can you separate out the rules and properties that actually need to differ between media queries, and those that should be common to all?